indico/modules/categories/controllers/display.py

```diff
@@ -620,21 +621,34 @@ def _process(self):
                             ongoing_event_count=len(ongoing_events),
                             ongoing_events_html=self._render_ongoing_events(ongoing_events))
 
+    def _find_nearest_category(self, category_chain):
+        for index, category_data in enumerate(category_chain):
+            if category_data['id'] == self.category.id:
+                if index == len(category_chain) - 1:
+                    return category_data
+                else:
+                    return category_chain[index + 1]
+        return {'id': 0, 'title': 'No category'}
```

Comment:
`0` is the root (Home) category, so not a good default one. however, you will never get an unlisted event in this view, so i don't think "No category" needs to be handled here (so i'd rather fail with an exception in that case)

Reply:
Fixed in [8a0afb7](https://github.com/indico/indico/pull/6110/commits/8a0afb730401a053eff6c593e5fb82a8d72ef430).